src/snowflake/snowpark/_internal/analyzer/analyzer.py

```diff
@@ -344,14 +345,17 @@ def table_function_expression_extractor(self, expr: TableFunctionExpression) ->
             sql = function_expression(
                 expr.func_name, [self.analyze(x) for x in expr.args], False
             )
-        elif isinstance(expr, NamedArgumentsTableFunction):
+        elif isinstance(expr, NamedArgumentsTableFunction) or isinstance(
```

Comment:
isinstance(expr, (NamedArgumentsTableFunction, GeneratorTableFunction))

Reply:
I was wondering if there was a concise way of doing this